app/containers/Sessions/containers/SettingsSessionPage/components/SessionSettings.js

```diff
@@ -204,6 +222,37 @@ const SessionSettings = ({
                 />
               </InputContainer>
 
+              {type === SessionTypes.SMS_SESSION && (
+                <InputContainer style={{ marginTop: 15 }}>
+                  <Row justify="between" align="center">
+                    <H3 mb={5} fontWeight="regular">
+                      {formatMessage(messages.smsCode)}
+                    </H3>
+                    <Text
+                      fontWeight="bold"
+                      color={themeColors.secondary}
+                      hoverDecoration="underline"
+                      clickable
+                      onClick={() =>
+                        editSession({
+                          smsCode: getRandomString(7),
+                        })
+                      }
+                    >
+                      Randomize code
+                    </Text>
```

Comment:
This should be a `button` but is a `p`. It affects accessibility. Replace it with button, try `TextButton` component.